### PR TITLE
fix: ranking and intent based response selector

### PR DESCRIPTION
**Pull request by dilyararimovna**

### CODE REVIEW

File: components/hE12LfxAkX3K9gU0nU4yE2.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: response_selector
-display_name: Response Selector
+display_name: Ranking-based Response Selector for Russian Language
```
Comment (smilni) [inline components/hE12LfxAkX3K9gU0nU4yE2.yml:2]: норм что не на русском? вроде для русских компонент хотели русские карточки

File: components/hjbdfiugef7h3niknto59u9dgf.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: response_selector
+display_name: Ranking- and Intent-based Response Selector for Russian Language
```
Comment (smilni) [inline components/hjbdfiugef7h3niknto59u9dgf.yml:2]: тоже

File: response_selectors/ranking_and_intent_based_response_selector/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Ranking- and Intent-based Response Selector
+
+## Description
+
+Response Selector is a component selecting final response among the given hypotheses by different skills.
+The Ranking- and Intent-based Response Selector utilizes floating point annotations by ranking hypotheses with 
+a candidate annotator (e.g., Sentence Ranker), scaling ranking scores with heuristics depending on entities and intents,
+and finally selecting the best ranked one.
```
Comment (smilni) [inline response_selectors/ranking_and_intent_based_response_selector/README.md:8]: Response Selector is a component that selects the final response among hypotheses provided by different skills.
The Ranking- and Intent-based Response Selector utilizes floating point annotations by ranking hypotheses with a candidate annotator (e.g., Sentence Ranker), scaling the resulting scores with heuristics based on entities and intents, and finally selecting the best ranked one.

File: components/hjbdfiugef7h3niknto59u9dgf.yml (hunk 2 above)
Comment (dilyararimovna) [reply components/hjbdfiugef7h3niknto59u9dgf.yml:2]: пойдет

File: components/hE12LfxAkX3K9gU0nU4yE2.yml (hunk 1 above)
Comment (dilyararimovna) [reply components/hE12LfxAkX3K9gU0nU4yE2.yml:2]: пойдет

**Pull request merged**